deepmd/utils/argcheck.py

```diff
@@ -718,9 +718,7 @@ def fitting_polar():
         Argument("resnet_dt", bool, optional=True, default=True, doc=doc_resnet_dt),
         Argument("precision", str, optional=True, default="default", doc=doc_precision),
         Argument("fit_diag", bool, optional=True, default=True, doc=doc_fit_diag),
-        Argument(
-            "scale", [List[float], float], optional=True, default=1.0, doc=doc_scale
-        ),
+        Argument("scale", List[float], optional=True, default=None, doc=doc_scale),
```

Comment:
Is there any reason to change it? The argument is exposed to users. Changing it will cause a breaking change.

Reply:
It's okay if `None` is equivalent to `1.0`, but it's a breaking change to disallow `float`.

Reply:
I see, I will update torch to match this behavior.